lib/datasource/go/index.spec.ts

```diff
@@ -132,6 +132,7 @@ describe('datasource/go', () => {
       expect(httpMock.getTrace()).toMatchSnapshot();
     });
   });
+
   describe('getReleases', () => {
```

Comment:
I'm confused as to what's going on with the tests for `getReleases`. All of them call `getPkgReleases` which is in `datasource/index.ts`. Shouldn't they be calling `getReleases` from `datasource/go/index.ts`?

Reply:
We intentionally pass all tests through the datasource index as it performs some additional massaging and we slowly push common functionality into the index.